manifests-maistra/1.1.0/servicemeshmemberrolls.crd.yaml

```diff
@@ -16,7 +16,22 @@ spec:
     status: {}
   version: v1
   additionalPrinterColumns:
-  - JSONPath: .spec.members
+  - name: Ready
+    description: How many of the total number of member namespaces are
+      configured
+    type: string
+    JSONPath: '.status.annotations.configuredMemberCount'
+  - name: Status
+    description: Whether all member namespaces have been configured or why
+      that's not the case
+    type: string
+    JSONPath: .status.conditions[?(@.type=="Ready")].reason
+  - name: Age
```

Comment:
How is this useful?  Would last modified be better, if you had this field at all?

Reply:
I added this mostly for consistency with all the other k8s resources, as they all have this column.